apps/dash-nlp/app.py

```diff
@@ -587,8 +596,103 @@ def plotly_wordcloud(data_frame):
     ),
 ]
 
+TOP_BIGRAM_PLOT = [
+    dbc.CardHeader(html.H5("Top bigrams found in the database")),
+    dbc.CardBody(
+        [
+            dcc.Loading(
+                id="loading-bigrams-scatter",
+                children=[
+                    dbc.Alert(
+                        "Something's gone wrong! Give us a moment, but try loading this page again if problem persists.",
+                        id="no-data-alert-bigrams",
+                        color="warning",
+                        style={"display": "none"},
+                    ),
+                    dbc.Row(
+                        [
+                            dbc.Col(["Choose a t-SNE perplexity value:"], md=6),
```

Comment:
```suggestion
                            dbc.Col(html.P(["Choose a t-SNE perplexity value:"]), md=6),
```


These two labels look non-consistent. 
**Before**: 
![Screen Shot 2020-03-30 at 11 12 06 AM](https://user-images.githubusercontent.com/35306149/77930134-7dc62100-7278-11ea-995c-264fbf5ba14a.png)

**After**: 

![Screen Shot 2020-03-30 at 11 21 54 AM](https://user-images.githubusercontent.com/35306149/77930277-b108b000-7278-11ea-8345-74a5cc953414.png)